typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts

```diff
@@ -156,9 +195,11 @@ async function checkBalance(
                 { ibcDenom: token.ibcDenom },
               );
               const tokenBalance = await adapter.getBalance(token.hypAddress);
-              return parseFloat(
-                ethers.utils.formatUnits(tokenBalance, token.decimals),
-              );
+              return {
```

Comment:
why not also do it for Cosmos?

Also wonder if we could pull the whole 
```
const balanceFloat = parseFloat(
                ethers.utils.formatUnits(balance, token.decimals),
              );

              const nativeValue = await getNativeTokenValue(
                chain,
                balanceFloat,
                tokenPriceGetter,
              );

              return {
                balance: balanceFloat,
                value: nativeValue,
              };
```
part out into a function or something, we duplicate that

same goes for the collateral stuff